internal/mode/static/state/dataplane/types.go

```diff
@@ -29,6 +29,8 @@ type Configuration struct {
 	HTTPServers []VirtualServer
 	// SSLServers holds all SSLServers.
 	SSLServers []VirtualServer
+	// TLSServers hold all TLSServers
+	TLSServers []Layer4Server
 	// Upstreams holds all unique Upstreams.
 	Upstreams []Upstream
```

Comment:
We will need separate upstreams for TLSServers because they are written to different contexts (http vs stream).